doc/developer-guide/consdbclient-summit-utils.rst

```diff
@@ -0,0 +1,5 @@
+############################
+ConsDbClient in summit_utils
+############################
+
+Is this a repeat, or a different scope than the other page?
```

Comment:
Actually, I could see pages in all *three* of:
* User Guide for querying using ConsDbClient
* Contributor Guide for inserting using ConsDbClient
* Developer Guide for how to write and test code in lsst-sitcom/summit_utils

Reply:
Oh that middle one is not a page, but it is a section in the Inserting page. I added the other sentences to the corresponding pages so it is more clear now.